README.md

```diff
@@ -9,7 +9,14 @@ A pallet for substrate based blockchains to enable the usage of smart contracts
 
 Smart contracts can directly be implemented and executed as Move scripts or modularized in Move modules. Therefor, the pallet supports publishing of Move modules and the execution of Move scripts to achieve this functionality. In the case of larger projects, the pallet provides the publishing of a bundle (multiple Move modules).
 
-For the execution of Move scripts and the publication of Move modules, the Move source code has to be compiled and serialized into bytecode. For this purpose, the tool [`smove`](https://github.com/eigerco/smove) is provided. The tool also provides further helpful features for developing and working with the Move language and this pallet.
+For the execution of Move scripts and the publication of Move modules, the Move source code has to be compiled and serialized into bytecode. For this purpose, the tool [`smove`][smove] is provided. The tool also provides further helpful features for developing and working with the Move language and this pallet.
+
+### Requirements
+
+- [Substrate developer environment](https://docs.substrate.io/install/)
+- [smove][smove] is a package manager for Move language in Substrate. Follow the instructions in its repo to install it.
+- [cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
```

Comment:
Mhh, so Docker is only necessary if you are willing to work with Docker - but the same could be true for cargo... If you directly use the Docker image, you could avoid installing cargo - couldn't you?

Reply:
Well, the `smove` installation requires `cargo` so 🤷